package.json

```diff
@@ -40,16 +40,16 @@
     }
   ],
   "dependencies": {
+    "@mojaloop/api-snippets": "^17.3.0-snapshot.0",
```

Comment:
Typically we don't check in snapshots, whats the blocker on those changes getting merged?

Everything else lgtm.

Reply:
Thanks @kleyow . Yes we are merging to feature branch. And the PRs on api-snippets, just got an update from Sam saying he is fine with the spec and we will merge them soon.